packages/verification/src/models/v1/sms-verification-report-response/sms-verification-report-response.ts

```diff
@@ -1,6 +1,5 @@
-import { VerificationStatusEnum } from '../enums';
+import { ReasonEnum, SourceEnum, VerificationStatusEnum } from '../enums';
 import { Identity } from '../identity';
-import { SMSVerificationReportResponsePrice } from '../verification-report-response-price';
 
 export interface SMSVerificationReportResponse {
```

Comment:
Is interface is related to https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications/#tag/Verifications/operation/ReportVerificationByIdentity!c=200&path=0/id&t=response ?
Online doc is listing only `id`, `method` and `status` field as response content.

Reply:
And these 3 fields are the one I got from the server. However, in the discussion with the Conversation team, it was mentioned that other properties can be part of this response: https://mailgun.slack.com/archives/C06R5UNSLR0/p1712846563563989